### DISCUSSION TITLE
I would like to volunteer to help edit/write the FAQ/Wiki/README

### DISCUSSION THREAD

**Discussion by Pmuzik** [linux-surface/linux-surface#1444] [Ideas]
Hi all, 

I've been quietly observing this repo for some time and I've noticed a lot of people tend to ask the same questions or types of questions rather frequently (classic problem, eh?)

I think the root of it is that the info presented is not easily digestible for the folks who most want to use Linux Surface. A lot of it comes across as giant walls of text. Unfortunately for you, a lot of people's brains (including mine) turn off when we see something visually overwhelming like the README. 

I would like to help out by collaborating with the engineers behind this project to better communicate what you (the programmers/engineers) want to communicate to your general users. I would be willing to go through the README, for instance, in little chunks and try to present it in a way that might transmit your message better. I suppose in a way I'm volunteering to be a community moderator (if you feel you need that).

This is something that I would be doing in my own free time, of course. And so, I might not be as quick to respond as I might want. Also, I'm still a student and I'll get very busy again once my fall semester starts. But I think I can help grease the wheels when it comes to troubleshooting between general users and the more technical folks.

Perhaps some of these condensed README/troubleshooting tips can be folded into the kernel as special man-pages.

What do ya'll think of this proposal?

Warmest regards,
Pat [:-)

**Comment by qzed** [comment]
Making the readme more readable/digestible sounds like a good idea. I'm not sure about the manpage thing (how many people, especially beginners, still use that?), but I think generally improvements regarding the document would be nice and I'd be happy if you want to take that up.

Let me know if you have any specific questions.

And don't worry about responsiveness, I think we're all doing this as a side project. As long as there is some communication it's all good.
